CHANGELOG.md

```diff
@@ -10,6 +10,13 @@ These are changes that will probably be included in the next release.
 ### Removed
 ### Fixed
 
+## [v0.4.14] - 2021-08-02
```

Comment:
Note: we already have https://github.com/timescale/timescaledb-docker-ha/releases/tag/v0.4.14